.circleci/config.yml

```diff
@@ -81,6 +81,14 @@ jobs:
       - run:
           name: Build Speech (C#)
           command: artman --api=speech_v1 --language=csharp --publish=noop
+      # TODO(jskeet): Remove C#/gRPC generation tests after Cshape gapic
+      # generation includes gRPC generation.
```

Comment:
What does that involve? Is there any particular benefit to combining two steps which use completely different generators?